.circleci/config.yml

```diff
@@ -49,10 +49,11 @@ jobs:
             - node_modules
 
       # prepare the database
+      - run: cp .env.testing .env
       - run: touch storage/testing.sqlite
-      - run: php artisan migrate --env=testing --database=sqlite_testing --force
+      - run: php artisan migrate
 
       # run tests with phpunit or codecept
-      #- run: ./vendor/bin/phpunit
-      - run: ./vendor/bin/codecept build
-      - run: ./vendor/bin/codecept run
\ No newline at end of file
+      - run: ./vendor/bin/phpunit
+      # - run: ./vendor/bin/codecept build
```

Comment:
what this for

Reply:
this line is commented, 
B.T.W This is codecept test runner we use phpunit not this package